Demos/Blazorise.Demo/Pages/Dashboard.razor

```diff
@@ -1,185 +1,109 @@
 ﻿@page "/"
+@using System.Dynamic
 @inject IVersionProvider VersionProvider
-
-<Heading Size="HeadingSize.Is1" Margin="Margin.Is3.OnY">
-    Blazorise UI Components for Blazor
-</Heading>
-
-<Lead Margin="Margin.Is4.OnY">
-    Blazorise is a component library built on top of Blazor and CSS frameworks like <Anchor To="https://getbootstrap.com/" Target="Target.Blank">Bootstrap</Anchor>, <Anchor To="https://tailwindcss.com/" Target="Target.Blank">Tailwind</Anchor>, <Anchor To="https://bulma.io/" Target="Target.Blank">Bulma</Anchor>, <Anchor To="https://ant.design/" Target="Target.Blank">Ant Design</Anchor>, and <Anchor To="http://daemonite.github.io/material/" Target="Target.Blank">Material</Anchor>. It can be used to build responsive, single-page web applications.
-</Lead>
+<Row>
+    <Column>
+        <DataGrid TItem="ExpandoObject"
+                  Data="inMemoryData"
+                  Responsive
+                  ShowPager
+                  ShowPageSizes
+                  Editable
+                  EditMode="DataGridEditMode.Inline"
+        NewItemCreator="NewItemCreator">
+            <DataGridAggregates>
+                <DataGridAggregate Field="@nameof( Employee.Email )" Aggregate="DataGridAggregateType.Count">
+                    <DisplayTemplate>
+                        @($"Total emails: {context.Value}")
+                    </DisplayTemplate>
+                </DataGridAggregate>
+                <DataGridAggregate Field="@nameof( Employee.Salary )" Aggregate="DataGridAggregateType.Sum" DisplayFormat="{0:C}" DisplayFormatProvider="@System.Globalization.CultureInfo.GetCultureInfo("fr-FR")" />
+                <DataGridAggregate Field="@nameof( Employee.IsActive )" Aggregate="DataGridAggregateType.TrueCount" />
+                <DataGridAggregate Field="@nameof( Employee.Childrens )" Aggregate="DataGridAggregateType.Sum" />
+            </DataGridAggregates>
+            <DataGridColumns>
+                <DataGridCommandColumn></DataGridCommandColumn>
+                <DataGridColumn Editable TextAlignment="TextAlignment.Center" Field="@nameof( Employee.Id )" Caption="#" Width="60px" />
+                <DataGridColumn Editable Field="@nameof( Employee.FirstName )" Caption="First Name">
+                </DataGridColumn>
+                <DataGridColumn Editable Field="@nameof( Employee.LastName )" Caption="Last Name" />
```

Comment:
Should we define RAW field names instead of using Employee?

Reply:
Yea... probably better so some devs aren't confused.